ArduSub/Sub.cpp

```diff
@@ -34,7 +34,7 @@ Sub::Sub()
           inertial_nav(ahrs),
           ahrs_view(ahrs, ROTATION_NONE),
           attitude_control(ahrs_view, aparm, motors, MAIN_LOOP_SECONDS),
-          pos_control(ahrs_view, inertial_nav, motors, attitude_control),
+          pos_control(ahrs_view, inertial_nav, motors, attitude_control, MAIN_LOOP_SECONDS),
```

Comment:
although the call to the attitude controller directly above it also uses MAIN_LOOP_SECONDS, sure this should use, "scheduler.get_loop_period_s()" shouldn't it?